regression/scripts/aerodynamics/aerodynamics.py

```diff
@@ -208,109 +205,26 @@ def reg_values():
     cd_m_ctrl_r     = np.array([ 0.0001,  0.0001,  0.0001,  0.0001,  0.0001,  0.0001,  0.0001,
                                  0.0001,  0.0001,  0.0001,  0.0001])
     
-    cd_p_fuse_r     = np.array([ 0.00581516 ,  0.00688254,  0.01062775,  0.00675354,  0.00682628,
-                                 0.00574868,  0.00707676, 0.00876415,  0.00686934,  0.00614271,
-                                 0.00718073])
+    cd_p_fuse_r     = np.array([ 0.00573221 ,  0.00669671,  0.01034335,  0.00656387,  0.00669973,
+                                 0.00560398,  0.00687499, 0.0085221,  0.00669252,  0.0060046,
+                                 0.00697051])
     
     cd_p_wing_r     = np.array([ 0.00568318,  0.00574321,  0.00911242,  0.00555159,  0.00636405,
                                  0.0048708 ,  0.00579879,  0.00734795,  0.00582637,  0.0054087,
                                  0.00583051])
     
-    cd_tot_r        = np.array([ 0.2595146 ,  0.05226547,  0.04064861,  0.02131243,
-                                 0.01927301,  0.01923316,  0.02958776,  0.048638,
-                                 0.08073247,  0.16186261,  0.11868474])
+    cd_tot_r        = np.array([ 0.25924093,  0.05201668,  0.04032639,  0.02111216,  0.01914386,
+                                 0.01908016,  0.02936072,  0.04833489,  0.08044945,  0.16153605,
+                                 0.11827559])
     
     
     
     return cd_c_r[:,None], cd_i_r[:,None], cd_m_r[:,None], cd_m_fuse_base_r[:,None], cd_m_fuse_up_r[:,None], \
            cd_m_nac_base_r[:,None], cd_m_ctrl_r[:,None], cd_p_fuse_r[:,None], cd_p_wing_r[:,None], cd_tot_r[:,None]
 
 if __name__ == '__main__':
-    #(conditions, configuration, geometry, test_num) = main()
+
     main()
     
     print 'Aero regression test passed!'
-    
```

Comment:
Oh I forgot to mention this. It's broken in the develop branch, and since the setup doesn't plot regression values I didn't think it was appropriate for a regression script. It could certainly be modified to do this, but that's not in the scope of this pull request. I can put it back if you want it to be fixed in another update.